pyqtgraph/graphicsItems/DateAxisItem.py

```diff
@@ -0,0 +1,215 @@
+import numpy as np
+import time
+from datetime import datetime
+from .AxisItem import AxisItem
+
+__all__ = ['DateAxisItem', 'ZoomLevel']
+
+MS_SPACING = 1/1000.0
+SECOND_SPACING = 1
+MINUTE_SPACING = 60
+HOUR_SPACING = 3600
+DAY_SPACING = 24 * HOUR_SPACING
+WEEK_SPACING = 7 * DAY_SPACING
+MONTH_SPACING = 30 * DAY_SPACING
+YEAR_SPACING = 365 * DAY_SPACING
+
+def makeMSStepper(stepSize):
+    def stepper(val, n):
+        val *= 1000
+        f = stepSize * 1000
+        return (val // (n*f) + 1) * (n*f) / 1000.0
+    return stepper
+
+def makeSStepper(stepSize):
+    def stepper(val, n):
+        return (val // (n*stepSize) + 1) * (n*stepSize)
+    return stepper
+
+def makeMStepper(stepSize):
+    def stepper(val, n):
+        d = datetime.utcfromtimestamp(val)
+        base0m = (d.month + n*stepSize - 1)
+        d = datetime(d.year + base0m / 12, base0m % 12 + 1, 1)
```

Comment:
`base0m / 12` will give `int` in Python 2 and `float` in Python 3.  
`base0m // 12` should work in both versions.